src/engine/Sprite.cxx

```diff
@@ -30,7 +30,18 @@ void Sprite::render() const
     {
       if (highlightSprite)
       {
-        SDL_SetTextureColorMod(m_SpriteData[currentLayer].texture, highlightColor.r, highlightColor.g, highlightColor.b);
+        if (SDL_SetTextureColorMod(m_SpriteData[currentLayer].texture, highlightColor.r, highlightColor.g, highlightColor.b))
+        {
+          // indicates failure.
```

Comment:
Put at least TODO, if error is not handled

Reply:
done